charts/refarch-templates/templates/configmap.yaml

```
@@ -6,7 +6,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "getFullname" $data }}
+  name: {{ $configmap.name }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)